modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php

```diff
@@ -106,6 +106,17 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#default_value' => $config->get('override_page_lang') ?? NULL,
     ];
 
+    $form['globan_settings']['sticky'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Sticky'),
+      '#options' => [
+        0 => $this->t('No'),
+        1 => $this->t('Yes'),
+      ],
+      '#description' => $this->t('Whether to make the banner sticky on top of the page or not.'),
+      '#default_value' => empty($config->get('sticky')) ? 0 : 1,
```

Comment:
Maybe we have to also don't use anymore empty() https://github.com/openeuropa/oe_webtools/pull/124/files#diff-b05913734899aef67f064ef0c9ade8b1R64 for consistency